### PR TITLE
add yaral syntax package

### PR DESCRIPTION
**Pull request by duongbkak55**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **
- [x] Any commands are available via the command palette.
- [x] Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
- [x]  If my package is a syntax it doesn't also add a color scheme. ***
- [x] I use [.gitattributes][3] to exclude files from the package: images, test files, sublime-project/workspace.

My package is highlighting for Yara L2 syntax, this was using for google cloud platform for create the rule.
My package is nearly to Yara Rule Syntax, but it should still be added because this is specific version of Yara into Yara L2 (https://cloud.google.com/chronicle/docs/detection/yara-l-2-0-overview), customize for using in google cloud rule.
Also, my package add some customize scope for clearly theme and special language.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

File: repository/y.json [hunk 1]
```diff
@@ -77,6 +77,17 @@
 				}
 			]
 		},
+		{
+			"name": "Yara L2 Syntax",
+			"details": "https://github.com/appliedsciencegroup/sublimetext-package-yaral2-syntax-highlighting",
+			"labels": ["Yara", "Yaral" ,"Yara L2", "syntax", "snippets"],
```
Comment (braver) [inline repository/y.json:83]: `"syntax"` -> `"language syntax"` is the canonical label.

**Comment by braver** [comment]
- Why does your package also include a color scheme? This is not necessary, if you follow the scope naming guide you never need to customize a color scheme to make it work.
- Your syntax alternates between `yara` and `yaral`, can you explain why?

**Comment by duongbkak55** [comment]
> * Why does your package also include a color scheme? This is not necessary, if you follow the scope naming guide you never need to customize a color scheme to make it work.
> * Your syntax alternates between `yara` and `yaral`, can you explain why?

* I want to addition some color for highlighting some special syntax. If this is a requirement condition for package, I can using existed scope in default color schema.
* About my syntax alternates between `yara` and `yaral`, sorry for my mistake. Because YaraL2 is nearly to Yara Rule Syntax, so I using some rule base from that without replacement. I will correct it.

**Comment by braver** [comment]
> If this is a requirement condition for package, I can using existed scope in default color schema.

Yes, it's unexpected for syntax package to require a specific color scheme (there are hundreds out there because people have their own preferences) or to modify any existing color scheme. For the same reason, plus people tend to use dozens of syntaxes.. you can imagine what a mess it would get if every syntax has its own opinion on the highlighting.

**Comment by braver** [comment]
Yes, that looks good, thanks. Please change the “syntax” label to “language syntax”, and we’re good to go.

File: repository/y.json (hunk 1 above)
Comment (duongbkak55) [reply repository/y.json]: I changed syntax label to language syntax.
Thanks

**Pull request closed without merging**